dandi/pynwb_utils.py

```diff
@@ -4,7 +4,7 @@
 from pathlib import Path
 import re
 import warnings
-
+from typing import List
```

Comment:
@Saksham20 I can tell that you're not using pre-commit properly.  You need to run `pre-commit install` in your local clone of the repository so that it will take effect automatically whenever you commit, and you also need to run `pre-commit run -a` again to enforce the changes that should have been made already.

Reply:
oops sorry, I pushed todays changes from another computer which did not have pre-commit. Pushing now